_data/cryptocurrencies.yml

```diff
@@ -2,231 +2,336 @@ websites:
-      doc: https://bitpay.com/two-factor
+      bcc: Yes
+      btc: Yes
+      othercrypto: No
+      doc: https://blog.bitpay.com/bitcoin-cash-wallet-beta/
+      exceptions:
+          text: "Bitpay now supports Bitcoin Cash recovery, along with storing and spending. Accepting BCC is not yet supported."
 
     - name: Bitsewa
       url: https://www.bitsewa.com
       img: bitsewa.png
-      tfa: Yes
-      software: Yes
-      doc: https://www.bitsewa.com/terms.php#2fa
+      bcc: No
+      btc: Yes
+      othercrypto: Yes
+      doc:
 
     - name: Bitstamp
       url: https://bitstamp.net
       img: bitstamp.png
-      tfa: Yes
-      software: Yes
-      doc: https://www.bitstamp.net/s/documents/bitstamp_2_factor_authentication_guide.pdf
+      bcc: Yes
+      btc: Yes
+      othercrypto: Yes
+      doc: https://www.bitstamp.net/article/bitcoin-cash-our-position/
 
     - name: Bittrex
       url: https://bittrex.com
       img: bittrex.png
-      tfa: Yes
-      software: Yes
-      doc: https://support.bittrex.com/hc/en-us/articles/115000198612-How-do-I-troubleshoot-backup-remove-disable-enable-Two-Factor-Authentication-2FA-
-
+      bcc: Yes
+      btc: Yes
+      othercrypto: Yes
+      doc: https://support.bittrex.com/hc/en-us/articles/115000808991-Statement-on-Bitcoin-Cash-BCC-
+      
     - name: Blockchain
       url: https://blockchain.info/
       img: blockchain.png
-      tfa: Yes
-      sms: Yes
-      email: Yes
-      software: Yes
-      hardware: Yes
-      doc: https://blockchain.info/wallet/support-pages
-
-    - name: BTCJam
-      url: https://btcjam.com/
-      img: btcjam.png
-      tfa: Yes
-      software: Yes
-      doc: http://help.btcjam.com/customer/portal/articles/1987686
-
+      bcc: Yes
+      btc: Yes
+      othercrypto: Yes
+      doc: https://blog.blockchain.com/2017/08/22/bitcoin-cash-update/
+      exceptions:
+          text: "Blockchain now has limited wallet support but does not yet offer full support."
+          
+    - name: Blockchair
+      url: https://blockchair.com/
+      img: blockchair.png
+      bcc: Yes
+      btc: Yes
+      othercrypto: No
+      doc: 
+      
+    - name: BTER
+      url: https://bter.com
+      img: bter.png
+      bcc: Yes
+      btc: Yes
+      othercrypto: Yes
+      doc: https://support.bittrex.com/hc/en-us/articles/115000808991-Statement-on-Bitcoin-Cash-BCC-
+      
     - name: CampBX
       url: https://campbx.com/
       img: campbx.png
-      tfa: Yes
-      software: Yes
-      doc: https://campbx.com/faq.php#security-compliance
+      bcc: No
+      btc: Yes
+      othercrypto: Yes
+      doc: 
 
     - name: CEX.IO
       url: https://cex.io/
       img: cex.png
-      tfa: Yes
-      sms: Yes
-      software: Yes
-      doc: https://cex.io/guide#4
+      bcc: Yes
+      btc: Yes
+      othercrypto: Yes
+      doc: https://blog.cex.io/news/bitcoin-cash-deposits-withdrawals-enabled-cex-io-16432
+
+    - name: Changally
+      url: https://shapeshift.io
```

Comment:
is changelly shapeshift?

Reply:
they're similar but competing products I believe. So yea, that needs fixing too

Reply:
I don't believe Changelly is Shapeshift. They're similar, but I am pretty sure they're separate entities.